### DISCUSSION TITLE
Sonoff dual R2 with Tasmota Fw --> control 2 relays with 1 button

### DISCUSSION THREAD

**Discussion by mrgiov** [arendst/Tasmota#20630] [Q&A]
Hi everybody,
I have a Sonoff Dual R2 with tasmota fw and I need to set this behavior:
with only one button I want to control both relays of modules like a common dual-state relay, so the behaviour will be:
--> 1°  press : relay n1 ON   --  relay n2 OFF
--->2° press:  relay n1 OFF --  relay n2 ON
--> 3° press:  relay n1 ON   --  relay n2 ON
--> 4° press:  relay n1 OFF  --  relay n2 OFF

How can I do this ? 

Thanks for everybody!

**Comment by sfromis** [comment]
With rules, you could react to number of times the button was pushed quickly, and issue power commands to match each state value.
https://tasmota.github.io/docs/Buttons-and-Switches/#multi-press-functions

**Reply by mrgiov** [reply]
Thanks but I never do that. Can u help me?

**Comment by mrgiov** [comment]
Yes but my request it's a little bit different because I want to do that
with single click as happen in a classic 2 way relay.

Il Mer 31 Gen 2024, 17:26 Barbudor ***@***.***> ha scritto:
